packages/server/lib/utils/tests.ts

```diff
@@ -78,7 +78,9 @@ export function apiFetch(baseUrl: string) {
 /**
  * Assert API response is an error
  */
-export function isError(json: any): asserts json is ApiError<any> {
+export function isError<TType extends Record<string, any>>(
+    json: TType extends { json: any } ? never : TType
+): asserts json is Extract<TType extends { json: any } ? never : TType, { error: any }> {
```

Comment:
is it possible to define the types in order to get rid of the duplication and make the signature a bit more readable: 
```
type T = Record<string, any>
type TT = T extends { json: any } ? never : T
...
```

Reply:
you can't define this types outside the function unless you make them generic, at least I didn't find a proper solution to make that better.